functions/pulsar-contributor-toolbox-functions.sh

```diff
@@ -695,3 +695,35 @@ function ptbx_show_latest_chart() {
   local chart=$2
   curl -s "${charturl}/index.yaml" | yq -o json eval |jq --arg chart "$chart" -r '.entries[$chart] | .[] | .version' | sort -n | tail -1
 }
+
+function ptbx_untar_latest_snapshot (){
+    (
+        ptbx_cd_git_root
+        local TARGET_DIR="./distribution/server/target"
+        local latest_generated_ss="$(ls -dt ${TARGET_DIR}/apache-pulsar-*-SNAPSHOT-bin.tar.gz | head -1)"
+        tar -xvf "$latest_generated_ss" -C "${TARGET_DIR}"
+    )
+}
+
+function ptbx_use_latest_snapshot_bin (){
+    {
+        ptbx_cd_git_root
+        local TARGET_DIR="distribution/server/target"
+        local latest_generated_ss="$(ls -dt ${TARGET_DIR}/apache-pulsar-*-SNAPSHOT | head -1)"
+        echo "Run the following to use the binary in the latest snapshot"
```

Comment:
btw. It's possible to change the current environment since the ptbx library  is loaded in the current shell.